include/deal.II/base/data_out_base.h

```diff
@@ -1725,6 +1725,175 @@ namespace DataOutBase
                        const std::vector<std_cxx11::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
                        const VtkFlags                          &flags,
                        std::ostream                            &out);
+  /**
+   * Some visualization programs, such as ParaView, can read several separate
+   * VTU files to parallelize visualization. In that case, you need a
+   * <code>.pvtu</code> file that describes which VTU files (written, for
+   * example, through the DataOutInterface::write_vtu() function) form a group.
```

Comment:
I think this description should have a big "You probably want to use ``DataOutInterface::write_pvtu_record`` rather than this function" note in the first paragraph.